presto-native-execution/presto_cpp/main/types/VeloxPlanConversion.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "presto_cpp/main/types/VeloxPlanValidator.h"
+#include "presto_cpp/presto_protocol/core/presto_protocol_core.h"
+#include "velox/common/memory/MemoryPool.h"
+
+namespace facebook::presto {
+
+protocol::PlanConversionResponse processVeloxPlanConversionRequest(
```

Comment:
We already have `PrestoToVeloxQueryPlan.h` so that seems a little confusing, shouldn't we try to differentiate from that by indicating this is to handle a request?

Reply:
Yeah, that's fair as well.

Its just that we usually prefer names that are not verbs.

Maybe validatePrestoToVeloxQueryPlan is good as well ? wdyt ?

Reply:
Yeah, that sounds fitting for the changes in this pr, but @rschlussel has plans to include the converted plan in the response to provide details for EXPLAIN. So it's planned to not just be a validation, which is why I didn't name it like that. @rschlussel do you have some thoughts on this?

Reply:
Ah I see... then could prestoToVeloxPlanConversion be an option ?

Reply:
that's ok I guess, I'll change it